src/tests/reviews.test.js

```diff
@@ -0,0 +1,106 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import cookieParser from 'cookie-parser';
+import server from '../index.js';
+
+chai.should();
+chai.use(chaiHttp);
+chai.use(cookieParser);
+
+const userDetails = {
```

Comment:
LGTM but one suggestion I have is to use a beforeEach hook to log in the user before executing the tests that require an authenticated user. This would make the test code cleaner and more readable. Additionally, it would prevent duplication of code by not having to send the login request in each test that requires it.